core/src/test/resources/newlines/alwaysBeforeTopLevelStatements.stat

```diff
@@ -124,7 +135,6 @@ object A {
 
   /** test */
   def foo(): Int = 0
-
```

Comment:
I expect this newline. A comment/scaladoc above a single-line makes it "multi-line", for me.